src/test/scala/io/iohk/ethereum/vm/CallOpcodesSpec.scala

```diff
@@ -100,15 +100,15 @@ class CallOpcodesSpec extends WordSpec with Matchers {
     private val stack = Stack.empty().push(if (op == DELEGATECALL) params.take(4) ++ params.drop(5) else params)
     private val mem = Memory.empty.store(UInt256.Zero, inputData)
 
-    val stateIn = ProgramState(context).withStack(stack).withMemory(mem)
-    val stateOut = op.execute(stateIn)
-    val world = stateOut.world
+    val stateIn: ProgramState[MockWorldState, MockStorage] = ProgramState(context).withStack(stack).withMemory(mem)
+    val stateOut: ProgramState[MockWorldState, MockStorage] = op.execute(stateIn)
```

Comment:
This is just for tests, honestly I like it better without explicit types. But if you have to, you can use [this](https://github.com/input-output-hk/etc-client/blob/phase/3/txExecution/src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala#L8) :)